packages/xod-project/test/patch.spec.js

```diff
@@ -1439,7 +1439,7 @@ describe('Patch', () => {
         });
         const res = Patch.computeVariadicPins(patch);
         assert.equal(res.isLeft, true);
-        Helper.expectErrorMessage(expect, res, MSG.patchHasNoVariadicMarkers('@/test'));
+        Helper.expectErrorMessage(expect, res, MSG.noVariadicMarkers('@/test'));
```

Comment:
For the test it would be *much* more readable if you’d compare to a real string literal rather than `MSG.someFuncWhoKnowsHowItWorks(...)`